ddl/ddl_api.go

```diff
@@ -1561,7 +1561,7 @@ func IsAutoRandomColumnID(tblInfo *model.TableInfo, colID int64) bool {
 	return false
 }
 
-func checkGeneratedColumn(ctx sessionctx.Context, colDefs []*ast.ColumnDef) error {
+func checkGeneratedColumn(ctx sessionctx.Context, schemaName string, tableName string, colDefs []*ast.ColumnDef) error {
```

Comment:
Maybe use CIString instead of string? (I'm OK with string, just curious which you consider is the better option here?)